### PR TITLE
Align with changes made in libusb1-sys

### PR DESCRIPTION
**Pull request by otavio**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
+matrix:
+  include:
+    - name: Linux with packaged libusb-1.0 library
+      os: linux
+      addons:
+        apt:
+          packages:
+            - libusb-1.0-0-dev
+      env:
+        - PKGCONFIG: libusb-1.0 libudev
```
Comment (a1ien) [inline .travis.yml]: I noticed that the real PKGCONFIG set only libusb-1.0.
https://travis-ci.com/a1ien/rusb/jobs/270644961#L187

Comment (otavio) [reply .travis.yml]: Fixed.

**Comment by a1ien** [comment]
I think if we use libusb from system, we not need libudev-dev. So we can remove them from check

**Comment by otavio** [comment]
You're right. I fixed this and did same change on `libusb1-sys`.

**Pull request merged**